network/encoding.go

```diff
@@ -252,8 +236,8 @@ func UnmarshalRegistered(buf []byte) (MessageTypeID, Message, error) {
 	return tID, ptrVal.Interface(), nil
 }
 
-// MarshalBinary the application packet => to bytes
-// Implements BinaryMarshaler interface so it will be used when sending with protobuf
+// MarshalBinary returns the bytes representation of the envelope's message and
```

Comment:
Thanks to your constant repetition, I think my head has spared some memory for that ... xD